src/main/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,64 @@
+package study;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    Pattern pattern = Pattern.compile("//(.)\n(.*)");
```

Comment:
자바에서는 상수를 표현할때 static final 접근 제어자를 선언합니다 😄 
```suggestion
    private static final Pattern pattern = Pattern.compile("//(.)\n(.*)");
```